modules/test/playwright/tests/frontend-data-set-views-web/dataSets.spec.ts

```diff
@@ -38,3 +38,25 @@ test('Assert table column labels', async ({dataSetsPage, page}) => {
 
 	await expect(tableColumnLabels).toEqual(expectedLabels);
 });
+
+test('Assert table action labels', async ({dataSetsPage, page}) => {
+	await dataSetsPage.goto();
+	await dataSetsPage.createDataSet();
+
```

Comment:
@kresimir-coko I've faced some issues with previous data set test. Mostly because the table is not loaded yet. I'm trying with `.waitFor` before the expect see [PR 4002](https://github.com/liferay-frontend/liferay-portal/pull/4002/files#diff-08a42956d167349a456aef18ae6b45478b56fc63e6fe7c17d2746a28dd67ca92R24)

Reply:
👍 ty @juanjofgliferay